src/authentication/helpers/get-profile-info.ts

```diff
@@ -11,6 +11,11 @@ function getUserInfo(): Avo.User.User | null {
 	return get(state, 'loginState.data.userInfo');
 }
 
+export const getFirstName = () => {
+	const state: any = store.getState();
```

Comment:
Could be wrong maar is dit niet van het `AppState` type? Lijkt me dat we deze `any` kunnen invullen.

Reply:
blijkbaar is het Reducer<AppState, AnyAction>